### ISSUE TITLE
Read Only not working

### ISSUE THREAD

**Issue by grivers3** [hassio-addons/addon-mqtt#14]
# Problem/Motivation

> "readonly": true, does not work

## Expected behavior

> if readony: true the user should not be able to write to topic

## Actual behavior

> can write to topic no matter readonly state

## Steps to reproduce

>    
```
   {"username": "hass",
      "password": "test",
      "readonly": false,
      "topics": [ "#"]},
    { "username": "sonoff_123456",
      "password": "test",
      "readonly": true,
      "topics": ["test/#"]}
```

docker exec -it addon_a0d7b954_mqtt tail /opt/acl
user hass
topic #
user sonoff_123456
topic readwrite test/#

## Proposed changes

> when "readonly": true 
acl should be
topic read [topic path]
when "readonly": false
would like to have an explixit readwrite
topic readwrite [topic path]
not
topic [topic path] - (Allows read & write)

**Comment by ludeeus** [comment]
I agrree...
<https://github.com/hassio-addons/addon-mqtt/commit/79a3e786007bee7f72a869c5aeb09b523b33279e> Should handle this

**Comment by ludeeus** [comment]
You can technically run it locally, but it's best to wait, will most likely push an update for this later this week

**Comment by ludeeus** [comment]
This is now included in 0.3.0, please update.
